ambari-server/src/main/resources/common-services/AMBARI_INFRA_SOLR/0.1.0/service_advisor.py

```diff
@@ -25,13 +25,19 @@
 SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
 STACKS_DIR = os.path.join(SCRIPT_DIR, '../../../stacks/')
 PARENT_FILE = os.path.join(STACKS_DIR, 'service_advisor.py')
+ABSOLUTE_STACKS_DIR = os.path.join(SCRIPT_DIR, '/var/lib/ambari-server/resources/stacks/')
```

Comment:
This feels dangerous.  The resources dir can be set in `ambari.properties` and can't be relied on like this.  You can resolve the "absolute" directory using python magic `os.path.abspath` with `STACKS_DIR`

Reply:
Yes, this will break anyone who uses this property (like I do).

Reply:
Fixed it.